environ_base.sh

```diff
@@ -41,8 +41,8 @@ export KOS_CPPFLAGS="${KOS_CPPFLAGS} ${KOS_INC_PATHS_CPP} -fno-operator-names"
 # Which standards modes we want to compile for
 # Note that this only covers KOS itself, not necessarily anything else compiled
 # with kos-cc or kos-c++.
-export KOS_CSTD="-std=c99"
-export KOS_CPPSTD="-std=gnu++98"
+export KOS_CSTD="-std=c11"
+export KOS_CPPSTD="-std=gnu++11"
```

Comment:
```suggestion
export KOS_CSTD="-std=c17"
export KOS_CPPSTD="-std=gnu++17"
```

After reading through the Defect Reports for 17 vs 11 I'd agree with your assessment and if we're going to move to 11 should move to 17 instead.

Tested this against the experimental toolchain as well as our legacy one. Found some other issues with newer code in the legacy toolchain (#579 and https://github.com/KallistiOS/kos-ports/pull/67 ) but none caused by this.